srcpkgs/kaffeine/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'kaffeine'
 pkgname=kaffeine
 version=2.0.18
-revision=2
+revision=3
 build_style=cmake
 configure_args="-DBUILD_TESTING=OFF"
 hostmakedepends="extra-cmake-modules gettext pkg-config kcoreaddons"
 makedepends="kidletime-devel kio-devel ksolid-devel vlc-devel
  qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql
  qt5-plugin-sqlite qt5-plugin-tds
  $(vopt_if dvb 'v4l-utils-devel eudev-libudev-devel')"
+depends="qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql
+ qt5-plugin-sqlite qt5-plugin-tds"
```

Comment:
Just wondering, if certain dependencies are in `depends`, is there any reason to have them in `makedepends` too (as is the case here), or is it always redundant to do so?